Makefile

```diff
@@ -0,0 +1,99 @@
+include Makefile-utils.mk
+
+#
+# Calypso
+#
+.PHONY: pre-start
+pre-start: welcom versions
+
+versions: B = $(FG_BLUE)
```

Comment:
This seems much less approachable than the old version check script, especially to a JS developer used to working with JS build systems. 🤷‍♂️

That might not be a reason not to go this route, but I just wanted to point it out.

Reply:
true but it also accomplishes in 9 lines what took 59 _and_ it removes a project dependency (`node_check_version`) _and_ it can operate in fewer shell invocations _and_ it will only run when `node` or `npm` actually change so it's not wasting cycles and introducing lag on every build :wink:

they are definitely tradeoffs, and I admit that it's not JavaScript, but I would like us all to stretch ourselves and look at it with an open mind: `make` is designed to efficiently build projects; Javascript is designed to run code in a browser.

Reply:
It's true that it's fewer lines of code, though I'd say the density/readability of the new 9 lines is a bit more intense.

Still, there are some cool benefits like you said. That's 👍